lib/matplotlib/font_manager.py

```diff
@@ -878,6 +878,10 @@ def set_size(self, size):
                     + ", ".join(map(str, font_scalings)))
             else:
                 size = scale * FontManager.get_default_size()
+        if size < 1.0:
+            _log.info('Fontsize %1.2f < 1.0 pt not allowed by FreeType. '
+                      'Setting fontsize = 1 pt' % size)
```

Comment:
You should not be doing the formatting, but passing a separate arg to `_log.info`, no?